scripts/path.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+# Delete the "patches" directory is
+[ -d "./patches" ] && rm -r patches
+
+# Go into the submodule directory and
+# generate the patch files
+cd ckeditor-dev
+
+git format-patch -1 liferay -o ../patches
```

Comment:
I don't think we want `-1` here but rather something that expresses "everything between the current merge base and the `HEAD` of the "liferay" branch", which would be something like:

```sh
git format-patch master...liferay -o ../patches
```

Reply:
That's exactly what I was looking for, thanks.